php/commands/server.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+class Server_Command extends WP_CLI_Command {
+
+	/**
+	 * Start a development server.
+	 *
+	 * ## OPTIONS
+	 *
+	 * --host=<host>
+	 * : The hostname to bind the server to. Default: localhost
+	 *
+	 * --port=<port>
+	 * : The port number to bind the server to. Default: 8080
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Make the instance available on any address (with port 8080)
+	 *     wp server --host=0.0.0.0
+	 *
+	 *     # Run on port 80 (for multisite)
+	 *     sudo wp server --host=localhost.localdomain --port=80
+	 *
+	 * @when before_wp_load
+	 * @synopsis [--host=<host>] [--port=<port>]
+	 */
+	function __invoke( $_, $assoc_args ) {
+		$min_version = '5.4';
+		if ( version_compare( PHP_VERSION, $min_version, '<' ) ) {
```

Comment:
Well, the `before_invoke` hook was [initially implemented][1] to avoid having to redefine a method in a child class. In this case, using it wouldn't bring much benefit.

[1]: https://github.com/wp-cli/wp-cli/commit/6ae19150fe3263b5b19e1237c58b92f73d9eab15